### DISCUSSION TITLE
Version 9.8.0.3

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#658] [Announcements]
<ul>
        <li>Added
            <ul>
                <li>Release notes since version 1.0</li>
                <li>Added option to set extension UI language</li>
                <li>A new way to translate <b>PDF</b>
                    <ol>
                        <li>Open website <a href="https://pdf.translatewebpages.org"
                                target="_blank">pdf.translatewebpages.org</a></li>
                        <li>Select or drag and drop a PDF file</li>
                        <li>The translation will start immediately and happen dynamically as you scroll through the page
                        </li>
                    </ol>
                </li>
                <li><b>Improve translation</b> button in page translation panel, with the options:
                    <ul>
                        <li>Select the original website language</li>
                        <li>Select target language</li>
                        <li>Page translation service</li>
                        <li>Do not sort translation results (This option will only apply to the current tab)</li>
                    </ul>
                </li>
                <li><b>Privacy</b> tab on the settings page, with the options:
                    <ul>
                        <li>Select the translation services you want to use</li>
                        <li>Silently use an alternate translation service if the current target language is not
                            supported by the selected translation service</li>
                    </ul>
                </li>
                <li><b>Experimental</b> tab on the settings page, with the options:
                    <ul>
                        <li>Configure your Libretranslate server for text (not page) translation.</li>
                        <li>Configure a DeepL Free API API key to translate text (not pages).</li>
                    </ul>
                </li>
                <li>Support for 26 new target languages:
                    <i>
                        Tibetan, Lower Sorbian, Emoji 😀 (yandex only), Fijian, Faroese, French (Canada), Upper Sorbian,
                        Inuinnaqtun,
                        Inuktitut, Inuktitut (Latin), Chinese (Literary), Mongolian (Traditional), Nyanja, Querétaro
                        Otomi, Dari,
                        Portuguese (Portugal), Rundi, Elvish (Sindarin), Serbian (Latin), Klingon (Latin), Klingon
                        (pIqaD), Setswana,
                        Tongan, Tahitian, Yucatec Maya, Cantonese (Traditional).
                    </i>
                </li>
                <li>Favorite languages in the language selector on Android</li>
            </ul>
        </li>
        <li>Changes
            <ul>
                <li>Nodes modified by <i>jiffyreader</i> will not be translated</li>
                <li>Links in Twitter posts will not be translated to improve translation quality</li>
                <li>When translating selected text on PDF and other restrictive websites, you now have the option to
                    listen to the original text as well</li>
                <li>Now tabs opened by TWP appear next to your current tab.</li>
                <li>When this version is installed, if your text translation service is set to DeepL, it will change to
                    Google or Bing</li>
            </ul>
        </li>
        <li>Fixed
            <ul>
                
                <li>Selected text translation not working in other extensions popup</li>
                <li>Layout breaks on some <i>Wikipedia</i> pages when the
                    <i><code>Do not sort translation results</code></i> option is enabled
                </li>
                
                <li>Pressing <b>Ctrl</b> twice in the selected text translation panel translates the entire page</li>
                <li>Comments section on sites like <b>Fox News</b> and <b>Yahoo news</b> are not translated</li>
            </ul>
        </li>
        <li>Bug fixes and general improvements</li>
        <li>Update interface translations</li>
    </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.8.0.3'>Version 9.8.0.3</a>.</em>

**Comment by dgknn89** [comment]
I loaded crx file in brave browser and got this error, I didn't get any error when I uploaded the zip file.

![Screenshot_2](https://github.com/FilipePS/Traduzir-paginas-web/assets/19433410/90eb166c-a14f-465c-a56f-ab16f0dc252a)

**Reply by FilipePS** [reply]
> I loaded crx file in brave browser and got this error, I didn't get any error when I uploaded the zip file.
> 

The **CRX** file is self-signed by me. To install it, you need to make a change to the Windows registry.

Detailed instructions are in the file [readme.md](https://github.com/FilipePS/Traduzir-paginas-web/blob/master/readme.md).

**Comment by ViktorKub** [comment]
Hello! Can you add a function to ignore the "newline" character when translating selected text? It would be very useful when translating documentation.
